elephant/spade.py

```diff
@@ -1265,6 +1265,10 @@ def pvalue_spectrum(spiketrains, binsize, winlen, dither, n_surr, min_spikes=2,
         if surr_method == 'joint_isi_dithering':
             surrs = [instance.dithering()[0] for
                      instance in joint_isi_instances]
+        elif surr_method == 'dither_spikes_with_refractory_period':
+            surrs = [surr.dither_spikes(
+                spiketrain, dither=dither, n=1, refractory_period=binsize)[0]
```

Comment:
`refractory_period=binsize` looks strange but who am I to question spade devs

Reply:
Since it is the initial refractory period, which then is corrected by the smallest ISI, I thought that it would be nice, to have the least value, that still keeps one spike per bin, for spike trains where anyway there are not two spikes close to each other.

Reply:
I also agree with Peter that this is an appropriate idea. In principle here our only worry is that the refractory period is at least one binsize, in order to prevent that we don't get spikes that end up very close to each other (in same bin).

Reply:
Peter, please put Alessandra's comment there.

Reply:
OK.